QUICKSTART-DEPLOY.md

```diff
@@ -7,18 +7,56 @@ This guide will help you get the Unkey deployment platform up and running locall
 - Docker and Docker Compose
 - Go 1.24 or later
 - A terminal/command line
+- dnsmasq (for wildcard DNS setup)
 
 ## Step 1: Start the Platform
 
-1. Start all services using Docker Compose:
+1. Set up the API key for ctrl service authentication by adding it to `go/apps/ctrl/.env`:
 
 ```bash
-docker-compose up metald-aio dashboard ctrl -d
+UNKEY_API_KEY="your-local-dev-key"
 ```
 
-2. Wait for all services to be healthy
+2. Set up dashboard environment variables for ctrl authentication in `apps/dashboard/.env.local`:
 
-The platform now uses a Docker backend that creates containers instead of VMs, making it much faster and easier to run locally.
+```bash
+CTRL_URL="http://127.0.0.1:7091"
```

Comment:
can you add these to the `pnpm local` command to set it automatically in the .env as well as the docker-compose dashboard app?